benchexec/tablegenerator/template.html

```diff
@@ -348,6 +354,7 @@
 
 $(document).ready(function() {
   $('body').keydown(hideContentPane);
+  $("#dataTable").floatThead({scrollingTop:'1.6em', useAbsolutePositioning:false, zIndex:3});
```

Comment:
Where this does value `1.6em` come from? The header is still not correctly vertically aligned, and I guess this is the problem.